src/plugins/plugin.h

```diff
@@ -283,6 +283,10 @@ struct janus_plugin {
 	 * @param[out] error An integer that may contain information about any error */
 	void (* const create_session)(janus_plugin_session *handle, int *error);
 	/*! \brief Method to handle an incoming message/request from a peer
+	 * @note The Janus core increases the references to both the \c message and \c jsep
+	 * json_t objects. This means that you'll have to decrease your own
+	 * reference yourself with a \c json_decref when you're done with them.
+	 * You'll also have to free \c transaction with \c g_free.
```

Comment:
Better remove the dot at the end, after `g_free`, since otherwise it will appear as part of the `code` block. Besides, we don't have it at the end of lines in most of the other doxygen comments.